src/client/eppo-client.ts

```diff
@@ -101,6 +103,9 @@ export type EppoClientParameters = {
   banditVariationConfigurationStore?: IConfigurationStore<BanditVariation[]>;
   banditModelConfigurationStore?: IConfigurationStore<BanditParameters>;
   configurationRequestParameters?: FlagConfigurationRequestParameters;
+  /**
+   * @deprecated obfuscation is determined by inspecting the `format` field of the UFC response.
+   */
```

Comment:
I'd add that "setting this field has no effect and its value will be ignored" to be extra clear

Reply:
Good idea. Added!